docs/testing.rst

```diff
@@ -49,3 +48,12 @@ to schedule requests or modify items returned from
 :meth:`~web_poet.pages.ItemPage.to_item`. You may need to adapt your spiders to
 this, for example checking if the special ``_SCRAPY_POET_SAVEFIXTURE`` setting
 is set to ``True`` and using more simple logic in this case.
+
+.. _fixture-adapter:
+
+Configuring the item adapter
+============================
+
+As documented in :ref:`web-poet:web-poet-testing-adapters`, fixtures can use custom
+item adapters. ``savefixture`` will use the adapter specified in the
+``SCRAPY_POET_TESTS_ADAPTER`` setting when saving the fixture.
```

Comment:
```suggestion
item adapters. ``savefixture`` command uses the adapter specified in the
``SCRAPY_POET_TESTS_ADAPTER`` setting to save the fixture.
```

Reply:
Mind that, if you append “command”, you need to prepend an article as well: `savefixture` → _the_ `savefixture` command.